v1.20/elastx/README.md

```diff
@@ -0,0 +1,13 @@
+# Conformance testing ELASTX Private Kubernetes
+
+## Ordering a cluster
+
+Get started by applying for a cluster at https://elastx.se/en/kubernetes.
```

Comment:
I tried to follow your instructions for ordering a cluster.

This link produced is insufficient. It's just a 302 redirect to your main website.

Nothing specific to Kubernetes here, or the steps necessary to (re)create a cluster.

```
curl -I https://elastx.se/kubernetes 
HTTP/2 301 
location: https://elastx.se/
server: nginx
date: Thu, 01 Jul 2021 00:24:23 GMT
content-type: text/html
content-length: 162
```

@amye was pointing to @taylorwaggoners's previous / similar request and Dan's [feedback on your 1.15 PR](https://github.com/cncf/k8s-conformance/pull/735#issuecomment-539976558).

She's noting that we've requested detailed instructions to reproduce the creation of the cluster before.

You did add instructions on how to run sonobouy for that PR, and I agree that a link is sufficient for that part, and is a common pattern.

However, all of your linked examples have much more detailed cluster setup information.

> the point of the directions is so that a customer can re-run the tests on their
own and confirm that your hosted platform still passes all the conformance tests. - Dan

Dan was requesting more verbose instructions for creating the cluster, which your current submission lacks.

More than happy to review updated instructions when provided!

Reply:
Thanks alot for saying saying what you want us to do :)

The reason for the redirect is that you missed /en in the url. However this URl is the product page and does not include an "Order Now" button. to make the process as easy as possibly I replaced the link with direct link to the order a product form.

Just let me know in case of any remaining questions regarding the ordering process.